docker/setup_wasmgovd.sh

```diff
@@ -6,6 +6,7 @@ STAKE=${STAKE_TOKEN:-ustake}
 FEE=${FEE_TOKEN:-ucosm}
 
 wasmgovd init --chain-id=testing testing
+sed -i "s/permission\": \"Everybody\"/permission\": \"Nobody\"/" "$HOME"/.wasmgovd/config/genesis.json
```

Comment:
Super nit, but wouldn't

`'s/permission": "Everybody"/permission": "Nobody"/'` 

be clearer (avoid the `\"`s)